tap_pardot/client.py

```diff
@@ -111,7 +119,7 @@ def _make_request(self, method, url, params=None):
 
     @backoff.on_exception(
         backoff.expo,
-        (PardotException),
+        (PardotException,Pardot5xxError),
         giveup=is_not_retryable_pardot_exception,
```

Comment:
The giveup function should be returning `False` if `isinstance(exc, Pardot5XXError)`